testsuite/tests/kuadrant/authorino/operator/tls/test_webhook.py

```diff
@@ -0,0 +1,135 @@
+"""
+Test raw http authorization interface.
+"""
+import base64
+import pytest
+from openshift import OpenShiftPythonException
+
+from testsuite.objects import Authorization
+from testsuite.certificates import CertInfo
+from testsuite.utils import cert_builder
+from testsuite.openshift.objects.validating_webhook import ValidatingWebhook
+from testsuite.openshift.objects.ingress import Ingress
+
+
+@pytest.fixture(scope="session")
+def specific_authorino_name(blame):
+    """Define specific name for authorino which matches with name in authorino certificate"""
+    return blame('authorino')
+
+
+@pytest.fixture(scope="session")
+def authorino_domain(openshift, specific_authorino_name):
+    """
+    Hostname of the upstream certificate sent to be validated by APIcast
+    May be overwritten to configure different test cases
+    """
+    return f"{specific_authorino_name}-authorino-authorization.{openshift.project}.svc"
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, authorino_domain, wildcard_domain):
+    """Certificate hierarchy used for the tests"""
+    chain = {
+        "envoy_ca": CertInfo(children={
+            "envoy_cert": None,
+            "valid_cert": None
+        }),
+        "authorino_ca": CertInfo(children={
+            "authorino_cert": CertInfo(hosts=authorino_domain),
+        }),
+        "invalid_ca": CertInfo(children={
+            "invalid_cert": None
+        })
+    }
+    return cert_builder(cfssl, chain, wildcard_domain)
+
+
+@pytest.fixture(scope="module")
+def authorino_parameters(authorino_parameters, specific_authorino_name):
+    """Setup TLS with specific name for authorino."""
+    authorino_parameters['specific_name'] = specific_authorino_name
+    yield authorino_parameters
```

Comment:
Why `yield` instead of `return` ?

Reply:
`yield` was there before, see https://github.com/Kuadrant/testsuite/blob/main/testsuite/tests/kuadrant/authorino/operator/tls/conftest.py#L160 but I've changed both.